Shoko.Server/API/v3/Controllers/PlexController.cs

```diff
@@ -0,0 +1,201 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Shoko.Server.API.Annotations;
+using Shoko.Server.API.v3.Plex;
+using Shoko.Server.Plex;
+using Shoko.Server.Settings;
+
+namespace Shoko.Server.API.v3.Controllers;
+
+[ApiController]
+[Route("/api/v{version:apiVersion}/[controller]")]
+[ApiV3]
+[Authorize]
+public class PlexController : BaseController
+{
+    private static string AlreadyAuthenticated = "User has already authenticated with plex yet. Invalidate the current token before re-authenticating.";
+
+    private static string NotAuthenticated = "User has not authenticated with plex yet. Authenticate first before retrying.";
+
+    private static string NoSelectedServer = "A server has not been selected yet. Select a server first.";
+
+    private static string InvalidServerSelection = "Invalid server selection.";
+
+    private static string InvalidLibrarySelection = "Selected libraries are invalid.";
```

Comment:
Use `const` for these, prevent overwriting them

Reply:
Will update it when i get home. 👍